### PR TITLE
fontawesome: revert to css

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description

- [x] update existing references & remove unused files 
- [x] replace CDN with local file
- [x] add some extra CSS to fix a few icon positing 

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)
- css

##### Testing
Tested it myself 

##### Fixes
- https://github.com/gitcoinco/web/issues/1331

### CODE REVIEW

File: app/dashboard/templates/shared/yge_auth.html [hunk 1]
```diff
@@ -28,30 +28,30 @@
   {{ user }}
 </div>
 {% else %}
-<script>
-  FontAwesomeConfig = { searchPseudoElements: true };
+<!-- <script>
+  // FontAwesomeConfig = { searchPseudoElements: true };
 </script>
-<script defer src="{% static "v2/js/fontawesome-all.min.js" %}"></script>
+<script defer src="{% static "v2/js/fontawesome-all.min.js" %}"></script> -->
```
Comment (mbeacom) [inline app/dashboard/templates/shared/yge_auth.html]: Should we drop this?

Comment (thelostone-mc) [reply app/dashboard/templates/shared/yge_auth.html]: derp. fixing it

File: app/dashboard/templates/shared/yge_auth.html [hunk 2]
```diff
@@ -28,30 +28,26 @@
   {{ user }}
 </div>
 {% else %}
-<script>
-  FontAwesomeConfig = { searchPseudoElements: true };
-</script>
-<script defer src="{% static "v2/js/fontawesome-all.min.js" %}"></script>
```
Comment (SaptakS) [inline app/dashboard/templates/shared/yge_auth.html]: Just curious. Don't you have to add the css link here as well like you did for `shared/head.html` ?

Comment (thelostone-mc) [reply app/dashboard/templates/shared/yge_auth.html]: Ew. I thought I added that

**Comment by thelostone-mc** [comment]
@mbeacom @SaptakS  : Done!

**Pull request merged**